pkg/cmd/release/download/download.go

```diff
@@ -78,10 +90,29 @@ func NewCmdDownload(f *cmdutil.Factory, runF func(*DownloadOptions) error) *cobr
 
 	cmd.Flags().StringVarP(&opts.Destination, "dir", "D", ".", "The directory to download files into")
 	cmd.Flags().StringArrayVarP(&opts.FilePatterns, "pattern", "p", nil, "Download only assets that match a glob pattern")
+	cmd.Flags().StringVarP(&opts.ArchiveType, "archive", "A", "", "Download default assset with specified type (zip or tgz)")
```

Comment:
1. For consistency with the web interface, let's use `tar.gz` instead of `tgz` since the former is the extension you would end up with if you've chosen to download a "tar.gz" from a release.
2. Any reason you preferred `-A` instead of `-a` as shorthand? I think either would be fine, but I'm interested in your thoughts.
3. Lets use `` `format` `` enclosed in backticks in the flag description because that renders nicely in the flag help docs.
```suggestion
	cmd.Flags().StringVarP(&opts.ArchiveType, "archive", "A", "", "Download the source code archive in the specified `format` (zip or tar.gz)")
```

Reply:
Used the ```-A``` just because some tools i've used before tend to use ```-a``` for something related to _all_ 😄 .
But we can go with the lowercase option if it is more consistent with the CLI pattern.